docs/cluster_spec.md

```diff
@@ -768,6 +768,33 @@ spec:
   sshKeyName: myexistingkey
 ```
 
+### Self-signed certificates towards Cloud APIs
```

Comment:
I think the heading should be the name of the field `mountCertificates`, defined prior to the additionals teps of adding `additionalUserData` to each `InstanceGroup`